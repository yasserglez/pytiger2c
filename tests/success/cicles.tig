for j:= 1 to 10 do
(
	for k:= 1 to 10 do
	(
		for l:= 1 to 10 do
			( printi(j*k*l); print(" ") );
		if k = 5 then break
	);
	if j = 5 then break
)