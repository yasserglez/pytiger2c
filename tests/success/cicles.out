1 2 3 4 5 6 7 8 9 10 2 4 6 8 10 12 14 16 18 20 3 6 9 12 15 18 21 24 27 30 4 8 12 16 20 24 28 32 36 40 5 10 15 20 25 30 35 40 45 50 2 4 6 8 10 12 14 16 18 20 4 8 12 16 20 24 28 32 36 40 6 12 18 24 30 36 42 48 54 60 8 16 24 32 40 48 56 64 72 80 10 20 30 40 50 60 70 80 90 100 3 6 9 12 15 18 21 24 27 30 6 12 18 24 30 36 42 48 54 60 9 18 27 36 45 54 63 72 81 90 12 24 36 48 60 72 84 96 108 120 15 30 45 60 75 90 105 120 135 150 4 8 12 16 20 24 28 32 36 40 8 16 24 32 40 48 56 64 72 80 12 24 36 48 60 72 84 96 108 120 16 32 48 64 80 96 112 128 144 160 20 40 60 80 100 120 140 160 180 200 5 10 15 20 25 30 35 40 45 50 10 20 30 40 50 60 70 80 90 100 15 30 45 60 75 90 105 120 135 150 20 40 60 80 100 120 140 160 180 200 25 50 75 100 125 150 175 200 225 250 